### ISSUE TITLE
Unable to sync using testnet

### ISSUE THREAD

**Issue by ghost** [nayutaco/NayutaCore#2]
I had no issues syncing nayuta on Mainnet, however when I changed the settings to sync on testnet and relaunched the app, it was not working as expected.

![Screenshot_20200901-021914__01](https://user-images.githubusercontent.com/13405205/91767956-8ee5d600-ebfa-11ea-919f-92cc596c069f.jpg)

![Screenshot_20200901-022230__01](https://user-images.githubusercontent.com/13405205/91767975-95744d80-ebfa-11ea-8a11-2fd2ef0b4863.jpg)

Logs: <pre><code>{"version":"0.10.2-beta.rc4 commit=v0.10.2-beta.rc4-dirty","commit_hash":"a37c3faab05dda1c4f9f7546e2f251b92d87d1f7","identity_pubkey":"0355405f8938a3ec51c0ef3b9e15f02caa2b3ec1eaf318f3369702ae31c4e1c39f","alias":"0355405f8938a3ec51c0","color":"#3399ff","num_pending_channels":0,"num_active_channels":0,"num_inactive_channels":0,"num_peers":0,"block_height":0,"block_hash":"000000000933ea01ad0ee984209779baaec3ced90fa3f408719526f8d77f4943","best_header_timestamp":"1296688602","synced_to_chain":false,"synced_to_graph":false,"testnet":true,"chains":[{"chain":"bitcoin","network":"testnet"}],"uris":[],"features":{"0":{"name":"data-loss-protect","is_required":true,"is_known":true},"5":{"name":"upfront-shutdown-script","is_required":false,"is_known":true},"7":{"name":"gossip-queries","is_required":false,"is_known":true},"9":{"name":"tlv-onion","is_required":false,"is_known":true},"13":{"name":"static-remote-key","is_required":false,"is_known":true},"15":{"name":"payment-addr","is_required":false,"is_known":true},"17":{"name":"multi-path-payments","is_required":false,"is_known":true}}} </code></pre>

I can also send debug.log file if you need it

**Comment by mandelmonkey** [comment]
Does it eventually start? I noticed tis but assumed it was just due to lack of neutrino nodes on testnet

**Comment by ghost** [comment]
No it didn't start even after waiting for 10-15 minutes and then I switched to Mainnet. Neutrino is used for LN node?

**Comment by mandelmonkey** [comment]
Yes it runs Neutrino for inital sync but uses bitcoind full node validation to verify neutrino is getting the correct blocks, I will take a look see if there is anything else breaking it

**Comment by mandelmonkey** [comment]
seems the seeds for bootstrapping the network on testnet are offline, we check if its the same for regular LND running on testnet with neutrino set (as nayuta core uses LND under the hood)
